index.html

```diff
@@ -0,0 +1,180 @@
+              <input type="email" class="form-control" id="formGroupEmailInput" placeholder="JaneSmith@JaneSmith.me">
+            </div>
+            <div class="form-group">
+              <label for="formGroupExampleInput">Phone Number</label>
+              <input type="number" class="form-control" id="formGroupPhoneInput" placeholder="(912) 555-4321"">
+            </div>
+            <div class="form-group">
+              <label for="formGroupAddressInput">Address</label>
+              <input type="text" class="form-control" id="formGroupAddressInput" placeholder="1234 Main St">
+            </div>
+          </div>
+          <div class="col right-col">
+            <div class="form-group">
+              <label for="formGroupExampleInput2">Professional Title</label>
+              <input type="text" class="form-control" id="formGroupLabelInput" placeholder="Software Developer">
+            </div>
+            <div class="form-group">
+              <label for="formGroupExampleInput2">Website</label>
+              <input type="text" class="form-control" id="formGroupWebsiteInput" placeholder="JaneSmith.me">
+            </div>
+            <div class="form-group">
+              <label for="formGroupExampleInput2">HeadShot</label>
+              <input type="text" class="form-control" id="formGroupPictureInput" placeholder="JaneSmith.jpeg">
+            </div>
+            <div class="form-group">
+              <label for="formGroupCityInput">City</label>
+              <input type="text" class="form-control" id="formGroupCityInput" placeholder="Oakland">
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="form-group padding col-md-6">
+            <label for="formGroupPostalCodeInput">Postal Code</label>
+            <input type="text" class="form-control" id="formGroupPostalCodeInput" placeholder="94115">
+          </div>
+          <div class="form-group padding col-md-2">
+            <label for="formGroupCountryCodeInput">Country Code</label>
+            <input type="text" class="form-control" id="formGroupCountryCodeInput" placeholder="US">
+          </div>
+          <div class="form-group padding col-md-4">
+            <label for="formGroupRegionInput">Region</label>
+            <input type="text" class="form-control"" id="formGroupRegionInput" placeholder="California">
+          </div>
+        </div>
+        <div class="mb-3">
+          <label for="validationTextarea">Summary</label>
+          <textarea class="form-control is-invalid" id="validationTextarea" placeholder="Please tell us a little about yourself. What do you want to tell employers about why you stand out." required></textarea>
+        </div>
+        <button type="submit" class="btn btn-primary">Submit</button>
+      </form>
+    </div>
+    <div class="dataContainer">
+      
+    </div>
+  </main>
+  <footer>
+
+  </footer>
+
+  <!-- JavaScript -->
+  <script src="./script.js"></script>
```

Comment:
Make sure you're loading your local JS last to avoid any issues with accessing libraries like jQuery.